src/ConcatenateMsg.chpl

```diff
@@ -27,14 +27,15 @@ module ConcatenateMsg
     proc concatenateMsg(cmd: string, payload: string, st: borrowed SymTab) : MsgTuple throws {
         param pn = Reflection.getRoutineName();
         var repMsg: string;
-        var (nstr, objtype, mode, rest) = payload.splitMsgToTuple(4);
-        var n = try! nstr:int; // number of arrays to sort
-        var fields = rest.split();
-        const low = fields.domain.low;
-        var names = fields[low..];
+        var msgArgs = parseMessageArgs(payload, 4);
+        var objtype = msgArgs.getValueOf("objType");
+        var mode = msgArgs.getValueOf("mode");
+        var n = msgArgs.get("nstr").getIntValue(); // number of arrays to sort
+        var names = msgArgs.get("names").getList(n);
+        const low = names.domain.low;
         
         cmLogger.debug(getModuleName(),getRoutineName(), getLineNumber(), 
-              "number of arrays: %i fields: %t low: %t names: %t".format(n,fields,low,names));
+              "number of arrays: %i low: %t names: %t".format(n,low,names));
```

Comment:
it doesn't look like `low` is used anymore other than in the debug statement, i think it should be safe to remove now

Reply:
Removed